packages/clarity-js/src/interaction/scroll.ts

```diff
@@ -50,8 +50,9 @@ function recompute(event: UIEvent = null): void {
 
     let current: ScrollState = { time: time(event), event: Event.Scroll, data: {target: element, x, y, top, bottom} };
 
+    // Commenting out this part to send the first scroll event for attention map
```

Comment:
lets not check in commented out code - if you dont need it just delete it please